pkgs/by-name/re/regolith/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "regolith";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "Bedrock-OSS";
+    repo = pname;
```

Do not copy `pname` to the `repo` attribute. Use a string literal.
```suggestion
    repo = "regolith";
```